app/inputs/multi_value_ordered_input.rb

```diff
@@ -29,7 +29,7 @@ def inner_wrapper(value, index)
   end
 
   def outer_wrapper
-    " <ul class='listing draggable-order dd-list' data-object-name='#{object_name}'>\n
+    " <ul id='ordered-creator' class='listing draggable-order dd-list' data-object-name='#{object_name}'>\n
```

Comment:
this shouldn't have an ID since it is intended to be a general reusable input

Reply:
👍